tools/oomph/openHAB-Core.setup

```diff
@@ -2368,16 +2368,16 @@
             pattern=".*/features/.*"/>
       </workingSet>
       <workingSet
-          name="Other">
+          name="Tests">
```

Comment:
Should we name it "integration tests" as the "unit tests" can now be placed in the bundle project itself?

Reply:
sure, I was not aware of this architectural change. Will update the PR.

Reply:
FYI: #482